app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    _, file_name, new_name = command.split()
+    try:
+        with open(file_name, "r") as file_in, open(new_name, "x") as file_out:
```

Comment:
The task description specifies that the function should do nothing if the source and destination file names are the same. You should add a condition to handle this case before attempting to open the files.